src/condor_utils/file_transfer.h

```diff
@@ -407,6 +408,7 @@ class FileTransfer: public Service {
 	FileTransferInfo Info;
 	PluginHashTable* plugin_table;
 	std::map<MyString, bool> plugins_multifile_support;
+	std::map<std::string, bool> plugins_from_job;
```

Comment:
I don't understand why this is `std::map<std::string, bool>` instead of `std::set<std::string>` since we appear to only check for existence.

Reply:
Eventually this will become a map to a class....